techniques.json

```diff
@@ -0,0 +1,162 @@
+[

```

Comment:
This is a duplicate techniques.json file that is in the main directory.

Reply:
Thanks for the help, first time doing this. I'm not entirely sure if I deleted the correct files, doI jusyt do in and trash the duplicates or start a new branch?

Reply:
You deleted the correct files, and did it right, but the issues in the first three review comments still are not resolved.